dev-cpp/string-theory/string-theory-3.3.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake ${GIT_ECLASS}
```

Comment:
${GIT_ECLASS} isn't defined here (anymore?)

Reply:
.... oversaw this.. removing.